### ISSUE TITLE
Feature Request: Emoji Eye candy

### ISSUE THREAD

**Issue by childnode** [obihann/archey-osx#40]
just an idea and just a draft: https://github.com/childnodeContribArch/archey-osx/commit/c9e8bc4a85be015d754634361e8d3245ef758eed

![bildschirmfoto 2015-12-02 um 20 15 48](https://cloud.githubusercontent.com/assets/192814/11541331/0f0317f6-9932-11e5-84a9-66b6b1f52383.png)

looking forward to get your feedback ;)

**Comment by obihann** [comment]
Very cute, I like it. But I think we should have this as an optional flag, what do you think?

**Comment by childnode** [comment]
yes, I think so too. we might evaluate a 3-way switch: text only, text and emoji, emoji only

**Comment by edrozenberg** [comment]
+1 for making emojis optional. Prefer the cleaner look without emojis personally and a way to turn them off is important.

CPU/Mem/Disk would need emojis also - any missing emojis look out of place (these 3 are missing in the screenshot above).

**Comment by childnode** [comment]
> CPU/Mem/Disk would need emojis also - any missing emojis look out of place (these 3 are missing in the screenshot above).

you're right, but that's because I had no idea what is a good emoji for these three things ;) Perhaps you have and we might add them?

a heart for CPU? ❤️
a MEMOry ... 📝
the closest it gets: we have a floppy(disk)💾
